community/broot/build

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+cargo build --release --locked
+
+install -Dm755 "target/release/broot" "$1/usr/bin/broot"
+
+# Build and install manpages.
+release_date=$(sed -n "s/.*v$2 - \(.*\)$/\1/p" CHANGELOG.md)
+cp man/page target/release/broot.1
+sed -i "s/#version/$2/g"         target/release/broot.1
+sed -i "s/#date/$release_date/g" target/release/broot.1
+install -Dm644 target/release/broot.1 "$1/usr/share/man/man1/broot.1"
```

Comment:
Change the `sed` lines to `man/page`, move the `install` line into the block of installs below

Reply:
TBH I don't remember why did I try to avoid in-place editing :shrug: 

> move the `install` line into the block of installs below

Together with completions?

Reply:
Yeah; it's just prettier that way :v